cmd/podman/containers/run.go

```diff
@@ -101,7 +103,19 @@ func run(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-
+	if rootless.IsRootless() {
+		if uid, err := strconv.ParseInt(cliVals.User, 10, 64); err == nil {
+			uids, _, err := rootless.GetConfiguredMappings()
+			if err != nil {
+				return err
+			}
+			for _, u := range uids {
+				if uid > int64(u.Size) {
+					return errors.Errorf("UID %d is greater then the range of UIDs available to the user. Need to modify /etc/subuid to allow this UID.", uid)
```

Comment:
Suggest `errors.Errorf("requested user's UID %d is too large for the rootless user namespace", uid)`